source/connect.txt

```diff
@@ -49,68 +49,69 @@ Connect
 
 When you open |compass-short|, an initial connection dialog appears:
 
-.. figure:: /images/compass/paste-connection-string.png
+.. figure:: /images/compass/new-connection.png
    :figwidth: 690px
-   :alt: Connection dialog for pasting your connection string
+   :alt: New connection dialog for pasting your connection string
 
-|compass-short| provides two methods to connect to your deployment,
-either providing your deployment connection string or filling in your
-deployment information in specific fields.
+|compass-short| provides multiple methods to connect to your deployment,
+either providing your deployment connection string or through Advanced 
+Connection Options. 
 
-.. tabs::
+Paste Connection String
+~~~~~~~~~~~~~~~~~~~~~~~
 
-   .. tab:: Paste Connection String
-      :tabid: paste-connection-string
+When you provide a connection string, |compass-short| supports
+most :manual:`Connection String Options
+</reference/connection-string/#connection-string-options>`
+supported by MongoDB.
 
-      When you provide a connection string, |compass-short| supports
-      most :manual:`Connection String Options
-      </reference/connection-string/#connection-string-options>`
-      supported by MongoDB.
+By default, |compass-short|'s default 
+:manual:`socketTimeoutMS </reference/connection-string/#urioption.socketTimeoutMS>`
+value is 60000, or 60 seconds. If you are frequently experiencing
+timeouts in |compass-short|, consider setting this option to a
+higher value in your connection string.
 
-      By default, |compass-short|'s default 
-      :manual:`socketTimeoutMS </reference/connection-string/#urioption.socketTimeoutMS>`
-      value is 60000, or 60 seconds. If you are frequently experiencing
-      timeouts in |compass-short|, consider setting this option to a
-      higher value in your connection string.
+For a complete list of the connection string options which
```

Comment:
I think "which" should be "that" in this case.

Reply:
Done.